wallet/src/error.rs

```diff
@@ -26,3 +26,11 @@ impl fmt::Display for Error {
         }
     }
 }
+
+impl Into<Value> for Error {
+    fn into(self) -> Value {
+        match self {
+            err => json!({ "code": 100, "message": format!("{}", err) }),
```

Comment:
```suggestion
            err => json!({ "code": self.code(), "message": format!("{}", err) }),
```

Instead of hardcoding the error codes as constants in a separate file, I would just write a error to code conversion function with a giant match of all the possible errors:

```rust
impl Error {
    fn code(&self) -> i64 {
        match self {
            Error::Execution(..) => 1
            _ => 100
        }
    }
}
```